lib/log.js

```diff
@@ -36,6 +40,10 @@ var levels = {
 
 _.each(levels, function(level, levelName) {
```

Comment:
I added a check before the method create part so that there will not be a log.silent() method.